test/FutureRideGeneratorTest.dart

```diff
@@ -0,0 +1,290 @@
+// This is a basic Flutter widget test.
```

Comment:
I'm a bit confused about how I can run these unit tests. Although I know your implementation generates recurring rides properly, would like to know how check these tests.

Reply:
I believe you run **flutter test test/FutureRideGeneratorTest.dart** in the terminal, let me know if that works!